src/cc_catalog_airflow/dags/provider_api_scripts/europeana.py

```diff
@@ -0,0 +1,282 @@
+
+Output:                 TSV file containing the images and the
+                        respective meta-data.
+
+Notes:                  https://www.europeana.eu/api/v2/search.json
+"""
+
+import argparse
+from datetime import datetime, timedelta, timezone
+import logging
+import os
+import re
+
+from common.requester import DelayedRequester
+from common.storage import image
+
+logging.basicConfig(
+    format='%(asctime)s - %(name)s - %(levelname)s:  %(message)s',
+    level=logging.INFO
+)
+
+logger = logging.getLogger(__name__)
+
+DELAY = 1.0
+RESOURCES_PER_REQUEST = '100'
+PROVIDER = 'europeana'
+API_KEY = os.getenv('EUROPEANA_API_KEY')
+ENDPOINT = 'https://www.europeana.eu/api/v2/search.json?'
+
+
+RESOURCE_TYPE = 'IMAGE'
+REUSE_TERMS = ['open', 'restricted']
+
+DEFAULT_QUERY_PARAMS = {
+    'profile': 'rich',
+    'reusability': REUSE_TERMS,
+    'sort': ['europeana_id+desc', 'timestamp_created+desc'],
+    'rows': RESOURCES_PER_REQUEST,
+    'media': 'true',
+    'start': 1,
+    'qf': [f'TYPE:{RESOURCE_TYPE}', 'provider_aggregation_edm_isShownBy:*'],
+}
+
+delayed_requester = DelayedRequester(DELAY)
+image_store = image.ImageStore(provider=PROVIDER)
+
+
+def main(date):
+    logger.info(f'Processing Europeana API for date: {date}')
+
+    start_timestamp, end_timestamp = _derive_timestamp_pair(date)
+    images_stored = _get_pagewise(start_timestamp, end_timestamp)
+
+    total_images = image_store.commit()
+    logger.info(f'Total images: {total_images}')
+    logger.info('Terminated!')
+
+
+def _get_pagewise(start_timestamp, end_timestamp):
+    cursor = '*'
+    total_number_of_images = 0
+    images_stored = 0
+
+    while cursor is not None:
+        image_list, next_cursor, total_number_of_images = _get_image_list(
+            start_timestamp,
+            end_timestamp,
+            cursor
+        )
+
+        if next_cursor is None:
+            break
+
+        cursor = next_cursor
+
+        if image_list is not None:
+            images_stored = _process_image_list(image_list)
+            logger.info(
+                f'Images stored: {images_stored} of {total_number_of_images}')
+
+        else:
+            logger.warning('No image data!  Attempting to continue')
+
+    return images_stored
+
+
+def _get_image_list(
+        start_timestamp,
+        end_timestamp,
+        cursor,
+        endpoint=ENDPOINT,
+        max_tries=6  # one original try, plus 5 retries
+):
+    for try_number in range(max_tries):
+
+        query_param_dict = _build_query_param_dict(
+            start_timestamp,
+            end_timestamp,
+            cursor
+        )
+
+        response = delayed_requester.get(
+            endpoint,
+            params=query_param_dict,
+        )
+
+        logger.debug('response.status_code: {response.status_code}')
+        response_json = _extract_response_json(response)
+        image_list, next_cursor, total_number_of_images = _extract_image_list_from_json(
+            response_json)
```

Comment:
```suggestion
        (
            image_list,
            next_cursor,
            total_number_of_images
        ) = _extract_image_list_from_json(response_json)
```
Please keep lines to 79 characters or less.